python/core/__init__.py.in

```diff
@@ -43,6 +43,9 @@ from .additions.runtimeprofiler import ScopedRuntimeProfileContextManager
 from .additions.validitycheck import check
 from .additions.ranges import datetime_range_repr, date_range_repr
 
+import numpy
```

Comment:
I'm not sure if numpy is a required dependency of QGIS python packages. Probably not. In that case it might be prudent to be robust to it missing

Reply:
Done. As per suggestion in the QEP.